code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -193,6 +193,9 @@
 	return 1 // update UIs attached to this object
 
 /obj/machinery/chem_dispenser/attackby(obj/item/reagent_containers/B, mob/user, params)
+	if(!anchored)
+		to_chat(user, "<span class='notice'>You must anchor [src] first!</span>")
+		return
 	if(isrobot(user))
```

Comment:
Robots will get the false hope now that they can interact with it once anchored. Evil man